src/main/java/coresearch/cvurl/io/request/RequestBuilder.java

```diff
@@ -139,7 +141,7 @@ public T logEnabled(boolean logEnabled) {
      * @return this builder
      */
     @SuppressWarnings("unchecked")
-    public T withProxy(CVurlProxy cVurlProxy) {
+    public T proxy(CVurlProxy cVurlProxy) {
```

Comment:
why u want to bear up backward compatibility?

Reply:
This is a new method, we don't need backward compatibility.